src/mlio/csv_record_tokenizer.h

```diff
@@ -55,13 +61,28 @@ class csv_record_tokenizer {
         return eof_;
     }
 
+    bool
+    is_truncated() const noexcept
+    {
+        return is_truncated_;
+    }
+
+private:
+    inline void
```

Comment:
You don't need the `inline` keyword in declarations of `reset()` and `push_char()`.